demisto_sdk/commands/format/format_module.py

```diff
@@ -405,3 +417,25 @@ def format_output(
         info_list.append(f"Format Status   on file: {input} - Success")
         info_list.append(f"Validate Status on file: {input} - Success")
     return info_list, error_list, skipped_list
+
+
+def is_graph_related_files(files: List[str], clear_cache: bool) -> bool:
+    """
+        Check if the files that Format should check are of type mapper, layout or incident fields.
+        Otherwise, we don't need to start the graph.
+
+        Args:
+            files (List[str]): a list of the paths of the files Format should check.
+            clear_cache (bool): wether to clear the cache.
+
+        Returns:
+            True if the files are of type mapper, layout or incident fields, else False.
+        """
+    for file in files:
+        file_path = file.replace("\\", "/")
```

Comment:
Why do we need this line?

Reply:
We need to update the path of the file to use \ instead of / . It is used  [here](https://github.com/demisto/demisto-sdk/blob/d310675950f17b526fa528aa0e2016eabc944da2/demisto_sdk/commands/format/format_module.py#L175C9-L175C9), and I used the same logic.

Reply:
Please replace it with a `str(Path(<the actual path string>))` which allows support for Windows paths (looks like that's what was attempted there), and more, and in a safer way.